### PR TITLE
Feat/market earn entrypoint

### PR DESCRIPTION
**Pull request by mitchellv-ledger**
### 📝 Description

Adds market entry point to wallet tab when ptxEarn feature flag is enabled.  Also adds feature flag capabilities to deeplinking.  Note deeplinking changes only come into effect when app is restarted.

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `LIVE-7451` 

### ✅ Checklist

- [] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 📸 Demo
Screenshot 2023-07-28 at 13 57 25

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by beths-ledger — APPROVED** [review]
Tested on ios and looks good 
image

Left one optional comment

**Review by gre — APPROVED** [review]

File: apps/ledger-live-mobile/e2e/models/wallet/walletTabNavigator.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import { getElementById, tapByElement } from "../../helpers";
-import { NavigatorName, ScreenName } from "../../../src/const";
+import { ScreenName } from "../../../src/const";
 
 export default class WalletTabNavigatorPage {
   nftGalleryTab = () => getElementById(`wallet-tab-${ScreenName.WalletNftGallery}`);
   portfolioTab = () => getElementById(`wallet-tab-${ScreenName.Portfolio}`);
-  marketTab = () => getElementById(`wallet-tab-${NavigatorName.Market}`);
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-mobile/e2e/models/wallet/walletTabNavigator.ts]: I think you could have just renamed the string in `getElementById` to `tab-bar-market` no?

File: apps/ledger-live-mobile/e2e/models/wallet/portfolioPage.ts [hunk 2]
```diff
@@ -43,4 +44,8 @@ export default class PortfolioPage {
   async openViaDeeplink() {
     await openDeeplink(baseLink);
   }
+
+  openMarketPage() {
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-mobile/e2e/models/wallet/portfolioPage.ts]: Given that this tab bar component is visible not just in the portfolio page I think you should have the action in the `walletTabNavigator.ts` below

Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/e2e/models/wallet/portfolioPage.ts]: Ah, I see the behaviour now if you enable/disable the feature flag. Fine to leave as is 👍

**Pull request merged**